NuGet.Config

```diff
@@ -4,7 +4,12 @@
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
   <packageSources>
+    <add key="LocalPackages" value="../LocalPackages" />
+    <add key="NuGet.org v3" value="https://api.nuget.org/v3/index.json" />
```

Comment:
Ordering the OD Dependencies feed before NuGet might be better. I did notice this is the same in the OD sln